R/age_group.R

```diff
@@ -0,0 +1,60 @@
+#' Create age groups
+#'
+#' @description
+#' A wrapper for \code{\link[base]{cut}} tailored for creating age groups. For
+#' each value of integer \code{x}, what age group does it fall into.
```

Comment:
I think I would re-word this slightly to something like "An extension of `cut()`. `age_group()` takes a numeric vector and assigns each age to the appropriate age group."